### ISSUE TITLE
Nintendont is just screwing with me.

### ISSUE THREAD

**Issue by ThePlushCollectorandRomHacker** [FIX94/Nintendont#1167]
After I recently updated nintendont it isn't reading or showing my .ciso files, which is strange because in past versions they were able to run .ciso files just perfectly and that was the only way to play gamecube games I didn't own
![17015000973536437321488885599245](https://github.com/FIX94/Nintendont/assets/152681142/86a48e0e-a7e2-49ae-91fc-d2e3672a98ad)
I've tried the main way for it to read games and I've tried it without the folders, and I don't know what to do anymore. Is there a way around this?

**Comment by Alvarden15** [comment]
The same is happening to me. I recently updated nintendont and now it no longer recognized the games folder (all of my games are in iso format). And everything inside the folder is in order.

**Comment by Alvarden15** [comment]
Yeah, apparently this is a bug with the latest version of Nintendont. I've just downgraded it to an old version and now it works fine, the folder is recognized correctly.

For those who are having the same issue: try downgrading Nintendont and test it.
